src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -4735,7 +4735,7 @@ bool Test(S1? s1)
             comp.VerifyDiagnostics();
 
             var expectedOperator = comp.GlobalNamespace.GetMember<NamedTypeSymbol>("S1").GetMembers(WellKnownMemberNames.EqualityOperatorName).
-                OfType<MethodSymbol>().Single(m => m.ParameterTypes[0] == m.ParameterTypes[1]);
+                OfType<MethodSymbol>().Single(m => m.ParameterTypes[0].TypeSymbol == m.ParameterTypes[1].TypeSymbol);
```

Comment:
Consider switching to `Equals` on `ParameterTypes[x]` directly.

Reply:
Done.